cico_build_deploy.sh

```diff
@@ -51,18 +51,13 @@ rm -rf ${TARGET_DIR}/
 #BUILD
 docker build -t ${BUILDER_IMAGE} -f Dockerfile.build .
 
-mkdir ${TARGET_DIR}/
-mkdir ${TARGET_DIR}/images
+mkdir -m 777 ${TARGET_DIR}/
+mkdir -m 777 ${TARGET_DIR}/images
 
 docker run --detach=true --name ${BUILDER_CONT} -t -v $(pwd)/${TARGET_DIR}:/${TARGET_DIR}:Z ${BUILDER_IMAGE} /bin/tail -f /dev/null #FIXME
 
-
 docker exec ${BUILDER_CONT} sh scripts/build_guides.sh
 
-#Need to do this again to set permission of images and html files
-chmod -R 0777 ${TARGET_DIR}/
```

Comment:
I believe this must stay in because `scripts/build_guides.sh` is run as root, and copies images in `html/` that subsequently cannot be deleted by the regular user.

Reply:
will re-add, doesn't hurt to have it

Reply:
@tradej updated.